### DISCUSSION TITLE
Need advice to buy right product.

### DISCUSSION THREAD

**Discussion by artcontr** [bitcraze/discussions#454] [Q&A] [answered]
`Hi.`
I want to make a setup with 5 crazifly.
Each one should have a led ring controlled remotely.
Each one will have a base with a contactless charger.
I only need the crazyfyl to go up vertically (Z axis). They should stay each one above their own base.
I need to control the Z axis remotely. But X and Y must be automatically calculated by the drone to stay above the base.
I need them to be working outside and go up as high as possible (10 / 15 meters high would be great).
I was thinking that each base could have 1 or more IR leds so the drone could always see it's own base and stay above it.
Here is a small drawing attached to help understand : https://ibb.co/7gSm5k0
I'm really looking forward to any idea or any suggestion to what i can use for my project.
Thank you very much in advance :-)

**Comment by krichardsson** [comment, marked as answer]
Hi, 
it sounds like an exciting project, but what you want to do is not easy, and I'm not even sure it is possible (with the Crazyflie).

First, the Crazyflie is mainly intended for indoor use. Of course it can be used outdoors as well but it does not have any protection for the outdoor environment. 

Looking at our decks, it is not possible to add a Qi charger deck and LED ring deck under the Crazyflie at the same time. This means that you might have to design your own deck that supports charging and LEDs. Perhaps you can skip the contact less requirement and come up with some way to charge through connectors on the pad?

The positioning will be a challenge. You could possibly use the Loco system (with some hacks), but it would not be precise enough to land on the pads and I think you would need some other solution close to the pads that is better. A lighthouse base station could help close to the ground, but I doubt it will work outside in daylight. Also, if you opt for multiple positioning technologies there is a risk you have to add many different sensors which will add weight.
One option is maybe to add small cameras to the pads that can be used to position the drone in X/Y? Loco could be used for the height perhaps? 

Obviously some of these ideas would require a fair amount of work and is not part of the default functionality.

If you implement this project, it would be interesting to hear about your solutions :-)

**Reply by artcontr** [reply]
Thank you so much for you answer. I'll make some test and get back to you if i make any progress.
